components/style/style_adjuster.rs

```diff
@@ -6,7 +6,10 @@
 //! for it to adhere to the CSS spec.
 
 use app_units::Au;
-use properties::{self, ComputedValues, StyleBuilder};
+use properties::{self, CascadeFlags, ComputedValues};
+use properties::{SKIP_ROOT_AND_ITEM_BASED_DISPLAY_FIXUP, StyleBuilder};
+#[cfg(feature = "gecko")]
```

Comment:
nit: Instead of conditionally importing at the top, could you import it inside of the function? That seems slightly cleaner, and less conditionals to care about.